code/game/area/Space Station 13 areas.dm

```diff
@@ -2582,6 +2582,22 @@ area/security/podbay
 /area/awaycontent/a30
 	icon_state = "awaycontent30"
 
+////////////////////////VR AREAS///////////////////////////////////
+
+/area/vr
+	name = "VR"
+	requires_power = 0
+	dynamic_lighting = 0
+	no_teleportlocs = 1

```

Comment:
It's going to be placed using the space allocation system - meaning, a new Z level with no teleport allowed